ext/bin/pdbbox-init

```diff
@@ -79,26 +79,35 @@ migrator_passwd="$(dd if=/dev/urandom bs=1 count=32 | base64)"
 
 # This code relies on echo being a function in bash, not a subprocess
 # and "here documents" to prevent passwords from being visible in
-# "ps", etc.
-
-(cd "$pdbbox"
- install -m 0600 /dev/null test-pass
- echo -n "$passwd" > test-pass
- install -m 0600 /dev/null test-pass-migrator
- echo -n "$migrator_passwd" > test-pass-migrator
- install -m 0600 /dev/null test-pass-admin
- echo -n "$admin_passwd" > test-pass-admin)
-
-(cd "$PGBOX"
- cat > pgpass <<-EOF
+# "ps", etc.  We also assume that mktemp perms are always go-wrx,
+# across all the relevant platforms.
+
+tmp_pass="$(mktemp "$tmp_dir/tmp-test-pass-XXXXXX")"
+chmod 0600 "$tmp_pass" # Not entirely safe, but expected to be redundant
```

Comment:
Why is the `chmod` necessary? It looks like mktemp already documents that it creates files with those permissions.